src/native-common/Image.tsx

```diff
@@ -101,20 +101,23 @@ export class Image extends React.Component<Types.ImageProps, Types.Stateless> im
         }
 
         const additionalProps = this._getAdditionalProps();
+        const extendedProps: RN.ExtendedImageProps = {
+            source: imageSource,
```

Comment:
`source` exists in `ImagePropsBase` https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/index.d.ts#L3503

Reply:
Yes, but it's a required (not optional) prop. If I move it out of here, TypeScript marks it as an error because I'm not specifying the source.